guava-gwt/pom.xml

```diff
@@ -375,7 +373,7 @@
             </goals>
             <configuration>
               <module>com.google.common.GuavaTests</module>
-              <includes>${gwt.test.include}</includes>
+              <includes>**/GwtTestSuite.java</includes>
               <logLevel>${gwt.logLevel}</logLevel>
               <mode>htmlunit</mode>
               <htmlunit>FF38</htmlunit>
```

Comment:
Oops: The move of `TestModuleEntryPoint` below (which I can't seem to comment on directly) shouldn't happen: This commit should include only the `pom.xml` changes.

The `TestModuleEntryPoint` move is happening because MOE is reading the latest copy of its config from head, and that version of the config no longer has special handling for `TestModuleEntryPoint`. Ideally, MOE would use the version of its config at CL 335869875 for this change and use the new version of its config only for later CLs.

I suspect that `guava-testlib` won't actually build successfully at this specific commit. If you're feeling ambitious, you could amend the commit to undo the `TestModuleEntryPoint` changes. But then you might have to amend the later commit that removes `TestModuleEntryPoint` to remove the copies in their corrected location.... And probably no one will ever care: It's not like we're going to cut a release at this specific commit. So I am mainly just noting this in case someone does discover the problem later.

Someone should migrate us to Copybara :) (which hopefully doesn't have this problem...).

Reply:
So you're saying this commit moves files to a bad location, but two commits later the files are deleted anyway?

I shall choose to look the other way, then.